crates/servicepoint_binding_uniffi/generate-bindings.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -e
+
+cargo build --release
+
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+TARGET_PATH="$(realpath "$SCRIPT_PATH"/../../target/release)"
+SERVICEPOINT_SO="$TARGET_PATH/libservicepoint_binding_uniffi.so"
+LIBRARIES_PATH="$SCRIPT_PATH/libraries"
+
+echo "Source: $SERVICEPOINT_SO"
+echo "Output: $LIBRARIES_PATH"
+
+BINDGEN="cargo run --features=uniffi/cli --bin uniffi-bindgen -- "
+BINDGEN_CS="cargo run --features=cs --bin uniffi-bindgen-cs -- "
+BINDGEN_GO="cargo run --features=go --bin uniffi-bindgen-go -- "
+COMMON_ARGS="--library $SERVICEPOINT_SO"
```

Comment:
remove checked-in code, at least for languages that are not tested yet and do not have a project file to reference